app/controllers/api/mixins/service_templates.rb

```diff
@@ -14,6 +15,16 @@ def order_service_template(id, data, scheduled_time = nil)
 
       private
 
+      def request_from_ui?
```

Comment:
Can you use `auth_mechanism` here instead? https://github.com/ManageIQ/manageiq-api/pull/488/files#diff-a7e4db9275b9f3963c7d8d24685f6bb8R6

Reply:
Didn't we determine that it would be better to verify the token instead of solely going on if it's an http basic request versus token to know if the request is coming from the UI? At least that was my understanding from the conversation at https://github.com/ManageIQ/manageiq-api/pull/476#issuecomment-425211696.

Maybe that method could be extended or used in another way, but I don't think simply changing to use `auth_mechanism` is the correct solution.